jax/_src/lax/slicing.py

```diff
@@ -370,7 +370,7 @@ class ScatterDimensionNumbers(NamedTuple):
       are the mirror image of `collapsed_slice_dims` in the case of `gather`.
     scatter_dims_to_operand_dims: for each dimension in `scatter_indices`, gives
       the corresponding dimension in `operand`. Must be a sequence of integers
-      with size equal to indices.shape[-1].
+      with size equal to `scatter_indices.shape[-1]`.
```

Comment:
In RST, single backticks render as itallic. We probably want double backticks
```suggestion
      with size equal to ``scatter_indices.shape[-1]``.
```

Reply:
Actually, since this is consistent with the rest of the docstring, we can probably leave it as-is.

Reply:
Although note that's true of this entire code block...